src/main/java/manufacturing/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,217 @@
+package manufacturing.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import manufacturing.dao.CarDao;
+import manufacturing.exception.DataProcessingException;
+import manufacturing.lib.Dao;
+import manufacturing.model.Car;
+import manufacturing.model.Driver;
+import manufacturing.model.Manufacturer;
+import manufacturing.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars "
+                + "(car_model, manufacturer_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, car.getModel());
+            preparedStatement.setLong(2, car.getManufacturer().getId());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
```

Comment:
As a suggestion - you can use car.setId `(resultSet.getObject("GENERATED_KEY", Long.class));` if you use MySql or enter a `columnLabel` if you use postgres in order to avoid ambiguity.

Reply:
okay. done